x/nft/types/keys.go

```diff
@@ -13,6 +13,9 @@ const (
 	// ModuleName is the name of the module
 	ModuleName = "nft"
 
+	// avoid conflicts with sdk nft module
+	ModuleNameAlias = "nft2"
+
 	// StoreKey is the default store key for NFT
 	StoreKey = ModuleName
```

Comment:
wouldn't there be a conflict on the store?

Reply:
as long as we don't enable the SDK module, it might be fine?

Reply:
but we will want to migrate to it / enable it?

Reply:
https://github.com/crypto-org-chain/chain-main/blob/master/doc/architecture/adr-004.md

Reply:
hmm, maybe do that in another PR: https://github.com/crypto-org-chain/chain-main/issues/829